external-crates/move/move-compiler/src/sui_mode/typing.rs

```diff
@@ -205,7 +290,14 @@ fn function(context: &mut Context, name: FunctionName, fdef: &mut T::Function) {
     }) {
         context.in_test = true;
     }
-    if name.0.value == symbol!("init") {
+    if let Some(acquire_loc) = acquires.values().next() {
+        global_storage_error(
+            context,
+            *acquire_loc,
+            format!("Invalid usage of global storage acquires"),
```

Comment:
Are there _any_ valid usages of global storage acquires?

Reply:
No, how would you rephrase this?

Reply:
I will attempt something, but let me know what you think :)

Reply:
"Global storage acquires are not allowed in sui mode"